lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingRemoteManager.cs

```diff
@@ -77,15 +76,15 @@ public WritableRemoteManager(IPAddress localAddress, int port, ITcpPortProvider
         /// <summary>
         /// Constructs a DefaultRemoteManager. Does not listen for incoming messages.
         /// </summary>
-        /// <param name="injector">The injector to pass arguments to incoming messages</param>
+        /// <param name="streamingCodec">Streaming codec</param>
         [Obsolete("Use IRemoteManagerFactory.GetInstance() instead.", false)]
-        public WritableRemoteManager(IInjector injector)
+        public StreamingRemoteManager(IStreamingCodec<T> streamingCodec)
```

Comment:
This class needs a constructor that uses `ITcpPortProvider` to be used by default by Tang. Otherwise, this will not work on clusters that restrict the port range available to containers.